src/olympia/reviewers/management/commands/auto_reject.py

```diff
@@ -77,6 +77,7 @@ def reject_versions(self, *, addon, versions, latest_version):
                     amo.LOG.REJECT_VERSION_DELAYED.id,
                 )
             )
+            .order_by('created')
```

Comment:
the `Meta` ordering is `-created` so changing `.last()` to `.first()` would do the same thing, right?  No preference is this is clearer though.

Reply:
Yes. I figured it would be less confusing to keep using `last()` and force the ordering since we do want the "last created".